WordPress/twentytwentyfive
# Fix: Color problems with the newsletter sign-up pattern

**Description**

Fixes https://github.com/WordPress/twentytwentyfive/issues/526

Update cover blocks to use contrast color for texts to work in different variations.

**Screenshots**

https://github.com/user-attachments/assets/d348bbf1-7b34-448a-af9d-a1eebca959c7



**Testing Instructions**

1. Create a page.
2. Insert the "Newsletter sign-up" pattern
3. Test the page with the different style variations.
4. Confirm that it looks good and the colors follow a11y guidelines.
